seed/views/v3/analyses.py

```diff
@@ -35,10 +35,12 @@ def list(self, request):
         organization_id = request.query_params.get('organization_id', None)
         property_id = request.query_params.get('property_id', None)
         analyses = []
-        for analysis in Analysis.objects.filter(organization=organization_id):
-            property_view_info = analysis.getPropertyViewInfo(property_id)
-            if property_id is not None and property_view_info["number_of_analysis_property_views"] < 1:
-                continue
+        if property_id is not None:
+            analyses_queryset = Analysis.objects.filter(organization=organization_id, analysispropertyview__property=property_id).distinct()
+        else:
+            analyses_queryset = Analysis.objects.filter(organization=organization_id).distinct()
+        for analysis in analyses_queryset:
+            property_view_info = analysis.getPropertyViewInfo(property_id=property_id if property_id is not None else None)
```

Comment:
I think we can drop this ternary, it seems redundant (if property_id is None, pass None; if property_id is not None, pass property_id)

Reply:
I wrote that thinking it would remove the entire arg, like a silly person.  Thanks for pointing it out!